lib/ffi-icu/time_formatting.rb

```diff
@@ -190,8 +190,9 @@ def set_date_format(localized, pattern_str)
         end
       end
 
-      def skeleton_format(pattern, locale)
-          pattern = UCharPointer.from_string(pattern)
+      def skeleton_format(skeleton_pattern_str, locale)
+          skeleton_pattern_ptr = UCharPointer.from_string(skeleton_pattern_str)
```

Comment:
Super small suggestion, but can we chop off the `skeleton_` part of the variable names here? :)

Reply:
Well the reason I named it `skeleton_` is to differentiate it with the `pattern_ptr` which is the _output_ of `getBestPattern`. 

so `skeleton` -> `getBestPattern` -> `pattern`

I felt that was the best way to disambiguate